packages/node/src/config/index.ts

```diff
@@ -34,11 +34,11 @@ export function parseConfig(configPath: string, secrets: Record<string, string |
   return { ...parsedConfig, ois };
 }
 
-export function getMasterKeyMnemonic(): string {
-  const mnemonic = process.env.MASTER_KEY_MNEMONIC;
+export function getMasterKeyMnemonic(config: Config): string {
+  const mnemonic = config.nodeSettings.airnodeWalletMnemonic;
   // The node cannot function without a master mnemonic
   if (!mnemonic) {
-    throw new Error('Unable to find MASTER_KEY_MNEMONIC from the environment. Please ensure this is set first');
+    throw new Error('Unable to find Airnode wallet mnemonic in the configuration. Please ensure this is set first');
```

Comment:
Is this possible? We are validating the config file so we expect mnemonic to be defined at this place.

Reply:
We're not validating the config file yet. I think we can remove it once the validator is integrated.